source/chapters/vinyl_control.rst

```diff
@@ -137,7 +137,9 @@ You can use any of the timecode media supported by Mixxx:
 +----------------------------------------+---------------------+
 | Traktor Scratch MK1 Vinyl              | Very high           |
 +----------------------------------------+---------------------+
-| Traktor Scratch MK2 Vinyl              | Not supported       |
+| Traktor Scratch MK2 Vinyl              | Initial support     |
```

Comment:
I agree, Experimental Support would be best

Reply:
Changed to "Experimental"